portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -270,54 +325,103 @@ const UsersList: React.VFC<UsersListProps> = function UsersList(props) {
     []
   );
 
+  const renderUserInfoCell = useCallback((item: UserListItem) => {
+    return <UserInfo item={item} />;
+  }, []);
+  const renderActionCell = useCallback(
+    (item: UserListItem) => {
+      let variant: "destructive" | "default";
+      let text = "";
+      if (item.deleteAt != null) {
+        variant = "default";
+        text = renderToString("UsersList.cancel-removal");
+      } else if (item.isDisabled) {
+        variant = "default";
+        text = renderToString("UsersList.reenable-user");
+      } else if (item.isAnonymized) {
+        variant = "destructive";
+        text = "";
+      } else if (item.anonymizeAt != null) {
+        variant = "destructive";
+        text = renderToString("UsersList.cancel-anonymization");
```

Comment:
@Ngkaokis Do you remember why you write L341 and L344 as 2 conditions?

Is there a possibility where `!isAnonymized && anonymizeAt != null`?

Reply:
Oh I see. When the anonymization is scheduled, but the scheduled time is yet to come